doc/template.txt

```diff
@@ -136,6 +136,10 @@ plug-in:
 		|template-user-variables| for more details.
 		(Default: `[]`).
 
+`g:templates_use_licensee`
+		Use licensee to detect project license when expanding
+		`%LICENSE%`. (Default: `1`)
```

Comment:
Here the documentation mentions that the default is `1`, but then below the code sets the default to `0`. Please make the documentation and the code agree on this :wink: